forge-gui/res/cardsfolder/upcoming/borborygmos_and_fblthp.txt

```diff
@@ -0,0 +1,16 @@
+Name:Borborygmos and Fblthp
+ManaCost:2 G U R
+Types:Legendary Creature Cyclops Homunculus
+PT:6/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ DBDraw | OptionalDecider$ You | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, draw a card, then you may discard any number of land cards. When you discard one or more cards this way, CARDNAME deals twice that much damage to target creature.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ DBDraw | TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, draw a card, then you may discard any number of land cards. When you discard one or more cards this way, CARDNAME deals twice that much damage to target creature.
+SVar:DBDraw:DB$ Draw | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | DiscardValid$ Land | NumCards$ X | Optional$ True | Mode$ TgtChoose | RememberDiscarded$ True | SubAbility$ TrigImmediateTrig
+SVar:TrigImmediateTrig:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ GE1 | Execute$ TrigDoubleDamage
```

Comment:
Trigger needs to Remember so you can Cleanup as Sub directly

Reply:
Check out cards like _Magma Pummeler_ for how to do it correctly

Reply:
So conditiondefined isn't needed ?

Reply:
Hmn better to keep it on this

Reply:
@tool4ever Hmm i've compared with cards like Magma Pummeler and I'm not seeing what currently wrong with the script ? Am I missing something ?

Reply:
The effect currently only works because you moved the Cleanup inside the second trigger
but that is wrong since that might get countered and then next time you might have two cards remembered and get a wrong result

Reply:
Oh I see, cleanup must happen after line 10 ?

Reply:
Well you shouldn't do it from inside the second trigger either because if that gets countered next time you would end up with two remembered and get a wrong result

The correct way is to transfer the Remembered onto the trigger just like you would do with an Effect

Reply:
``NumDmg$ Count$TriggerRememberAmount`` or it's still wrong

Reply:
Sorry I think it's fine now.

Reply:
Yea looks right finally :P